pkg/cso/v1/validator.go

```diff
@@ -187,11 +188,15 @@ var cloudProviderRegions = map[string]types.StringArray{
 		"brazilsoutheast",
 	},
 	"azure-us-gov": {
+		"global",
 		"usgovvirginia",
 		"usgoviowa",
 		"usgovarizona",
 		"usgovtexas",
 	},
+	"global": {
```

Comment:
You will have a secret addressed by `infra/global/<account>/global/<service>/<suffix>` path which can be interpreted as provider / region unbound secret, which is the definition of a platform secret in CSO.

From a security point of view, having a shared secret across all infra is not correct. 

As a reminder :

`infra` ⇒ Infrastructure secrets tightly coupled to infrastructure provider

<img width="720" alt="Capture d’écran 2020-12-17 à 07 28 42" src="https://user-images.githubusercontent.com/3110/102451810-86929900-4039-11eb-9cd3-5b71f5326ac2.png">

`platform` ⇒ Logical infrastructure (spread over multiple infra)

<img width="657" alt="Capture d’écran 2020-12-17 à 07 29 13" src="https://user-images.githubusercontent.com/3110/102451854-95794b80-4039-11eb-90d3-f56efe648106.png">

I suggest you to migrate this secret in the platform ring.

`platform/production/<platform-name>/global/<component>/<suffix>`

Visually the infra / platform distribution could be drawn like the following diagram :

![CSO_RING_1_2](https://user-images.githubusercontent.com/3110/102455021-29014b00-403f-11eb-9626-b5f91bd13b49.png)

And Platform / Product / App relations

![CSO_3_4](https://user-images.githubusercontent.com/3110/102456924-3835c800-4042-11eb-849b-5847f97dd8db.png)



CC @elastic/cloud-security

Reply:
I think I get it, let me make the proper adjustments.

Reply:
OK, removed `infra/global` in latest changeset.